mrgeo-cmd/mrgeo-cmd-core/src/main/java/org/mrgeo/cmd/MrGeo.java

```diff
@@ -141,33 +141,68 @@ public int run(String[] args) throws IOException
     loadCommands();
   }
 
+  Options options = createOptions();
   if (args.length == 0)
   {
-    usage();
+    usage(options);
     return -1;
   }
 
-  Options options = createOptions();
+  String cmdStr = args[0];
+  if (cmdStr.equals("-h")) {
```

Comment:
Thanks for taking a look. Good point in general, but in this particular case it already checks the args.length before that, so args[0] can't be null.